Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs

```diff
@@ -78,70 +77,61 @@ private static string ConvertDataSetToJson(DataSet result, Options options, stri
                 continue;
```

Comment:
I don't think this if-statement actually does anything, so it can be removed.

Reply:
Why do you think that this does not do anything though? It checks if we want to export a single sheet and if we are looking at a sheet different from that - we skip it.